lib/p5.play.js

```diff
@@ -1356,8 +1356,8 @@ function Sprite(pInst, _x, _y, _w, _h) {
     //if has animation get the animation bounding box
     //working only for preloaded images
     if(animations[currentAnimation]) {
-      this._internalWidth = animations[currentAnimation].getWidth()*abs(this._getScaleX());
-      this._internalHeight = animations[currentAnimation].getHeight()*abs(this._getScaleY());
+      this._internalWidth = animations[currentAnimation].getWidth();
+      this._internalHeight = animations[currentAnimation].getHeight();
       this.setCollider('rectangle');
     } else {
       this.setCollider('rectangle');
```

Comment:
All tests pass by removing these lines, and a quick spot check of p5.play examples and a gamelab example, this seems to work.